espanol/rezando/enano/enano.md

```diff
@@ -1,3 +1,5 @@
+Acto seguido el enano golpeó detrás de las piernas a Jurema, que calló al suelo de rodillas.
 Sin permitir hablar a Jurema el enano habló: "Soy Tuleto y he venido a iluminar sus mentes con la luz del ..."
 
-[Amo del fuego](fuego/fuego.md)
\ No newline at end of file
+[Amo del fuego](fuego/fuego.md)
+[Ombligo de Cacatucán](ombligo/ombligo.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.